dev/core_generator/lib/src/definition.dart

```diff
@@ -30,6 +30,8 @@ class Definition {
   Key _key;
   bool _isAbstract = false;
   bool _editorOnly = false;
+  bool _forRuntime = true;
```

Comment:
Can these 'runtime' changes land on their own in a separate PR?

Reply:
Changes here: https://github.com/rive-app/rive-cpp/pull/182